src/common/NoticeBox.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+import { logger } from '@nordicsemiconductor/pc-nrfconnect-shared';
+
+import { DevZoneLink } from './Link';
+
+export default ({
+    mdiIcon,
+    color,
+    title,
+}: {
+    mdiIcon: string;
+    color: string;
+    title: string;
+}) => (
+    <div className="tw-flex tw-flex-row tw-items-center tw-gap-4 tw-bg-gray-50 tw-p-4 tw-text-gray-700">
+        <span
+            className={`mdi ${mdiIcon} ${color} tw-align-middle tw-text-4xl/9`}
+        />
+        <div>
+            <b>{title}</b>
+            <p>
+                Contact support on <DevZoneLink /> if the problem persists.
```

Comment:
I would include the log in the notification. Including it _before_ DevZone will trigger the user to check it first, and only then contact DevZone. This can save some time to our Tech Support.

Suggestion, which needs some code changes:

```suggestion
                Check <log> and contact support on <DevZoneLink /> if the problem persists.
```

Reply:
Should have mentioned this earlier, but this log button does not open the log file contents (like the `open log` button does in our other apps). Here it opens the file explorer with the current log pre-selected so that they only have to drag and drop it into the support ticket (was my thought at least).

Reply:
I don't think what you say really contradicts my proposal. It just means one extra click to open, and the value of having the log mentioned before DevZone is in my opinion higher than whether <log> opens the log or points to its path :)

Reply:
I just wanted to mention it since it wasn't obvious ;)

Reply:
Up to you. I know that the first answer to most of the support tickets is "can you share the log", so I tried to address that. My issue is that the log is hanging at the end of the sentence without a clear purpose. An alternative would be to keep the phrasing of the line, but move the log with a </br> to the next line.

Reply:
Another option:
```suggestion
                Contact support on <DevZoneLink /> and provide <log> if the problem persists.
```